File: languages/java/testData/excludeFromInspection/notExclude.java
package testData.excludeFromInspection;

public class Fooo {
    public void boo(int b<caret>oi) {
        var s<caret>ch = getSomeSchoolNumber();
        try {
            System.out.println("Oh My!");
        } catch (Exception ignore) {
        } catch (Exception ignored) {
        } catch (Exception <caret>x) {
        }
    }

    private int getSomeSchoolNumber() {
        return 239;
    }
}

class Fooo2 extends Fooo {
    @java.lang.Override
    public void boo(int boi){
        System.out.println("Hello World!");
    }
}

File: languages/kotlin/testData/excludeFromInspection/notExclude.kt
package excludeFromInspection

open class Foo3 {
    open var xy = 0
    open fun boo(b<caret>oi: Int) {
        val sc<caret>h = getSomeSchoolNumber()
        try {
            println("Oh My!")
        } catch (ignore: Exception) {
        } catch (ignored: Exception) {
        } catch (_: Exception) {
        } catch (<caret>e: Exception) {
            println("Exception is not ignored")
        }
    }

    private fun getSomeSchoolNumber(): Int {
        return 239
    }
}

internal class Foo4 : Foo3() {
    override var xy = 1
    override fun boo(boi: Int) = println("Hello World!")
}
